### PR TITLE
Add support for ibazel_short_circuit tag

### PR DESCRIPTION
**Pull request by mrmeku**
Fixes https://github.com/bazelbuild/bazel-watcher/issues/333

### CODE REVIEW

File: ibazel/ibazel.go [hunk 1]
```diff
@@ -339,26 +353,31 @@ func verb(s string) string {
 }
 
 func (i *IBazel) build(targets ...string) (*bytes.Buffer, error) {
-	b := i.newBazel()
+	if i.bazel != nil {
```
Comment (achew22) [inline ibazel/ibazel.go:356]: This kills the `i.bazel` indiscriminately, should there be a check to make sure short circuiting is enabled?

File: process_group/process_group_windows.go [hunk 2]
```diff
@@ -41,6 +41,10 @@ func (pg *winProcessGroup) RootProcess() *exec.Cmd {
 	return pg.root
 }
 
+func (pg *winProcessGroup) Run() *exec.Cmd {
```
Comment (achew22) [inline process_group/process_group_windows.go:44]: The type in `process_group_unix.go` and in `process_group_windows.go` do not match,

```
func (pg *unixProcessGroup) Run() error {
```

vs

```
func (pg *winProcessGroup) Run() *exec.Cmd {
```

File: ibazel/ibazel.go (hunk 1 above)
Comment (mrmeku) [reply ibazel/ibazel.go:356]: done, thanks for catching that!

File: process_group/process_group_windows.go (hunk 2 above)
Comment (mrmeku) [reply process_group/process_group_windows.go]: done

**Pull request closed without merging**